tests/test_interfaces.py

```diff
@@ -0,0 +1,779 @@
+# pylint: skip-file
+""" This module tests the interface module """
+
+import unittest
+import mock
+import wifiphisher.common.interfaces as interfaces
+import pyric
+
```

Comment:
You will have to set your path accordingly to execute the tests from whatever location. See how I did it in my tests.